L1Trigger/L1TTrackMatch/interface/L1TrackJetProducer.h

```diff
@@ -1,21 +1,18 @@
-#ifndef L1Trigger_L1TTrackMatch_L1TrackJetProducer_HH
```

Comment:
Why did you remove the header guard and replaced it with `#pragma once`?
I'm asking because it is not the usual way to deal with it in our code...

Reply:
In any case, this file only contains definitions for structs and a few inline functions, and it does not define the L1TrackJetProducer (correctly defined in L1Trigger/L1TTrackMatch/plugins/L1TrackJetProducer.cc instead).
As of now, this file is only included by L1TrackJetProducer.cc: if it has to remain as such, it could more conveniently be inserted therein. As far as I understand you are expecting some wider usage for the struct and inline functions defined here: if so, please rename the file in order to make it clear that this is not the header file with the interface for the L1TrackJetProducer

Reply:
Done